modules/apps/portal/portal-cluster-multiple/src/main/java/com/liferay/portal/cluster/multiple/internal/ClusterExecutorImpl.java

```diff
@@ -196,14 +192,18 @@ public boolean isEnabled() {
 	}
 
 	@Activate
-	protected void activate(ComponentContext componentContext) {
+	protected void activate(
+		BundleContext bundleContext, Map<String, String> properties) {
```

Comment:
@W2jx Why do you need to convert the component context parameter to bundle context + properties map? If it's really needed, please do it in a separate commit.